### ISSUE TITLE
Pagination Issue

### ISSUE THREAD

**Issue by dwthomson** [pods-framework/pods#5184]
I am using pagination, but the number 1 appears when there is only one page, any way to stop that behaviour?

**Comment by jimtrue** [comment]
I can verify this is reproducible. If you have 3 items and you set pagination="true' in your shortcode (or regular code). you always get a 1 at the bottom of the page.

This is what is generated by our code:
```
<p><span class="pods-pagination-advanced ">	
	<span class="pods-pagination-number pods-pagination-current ">1</span>
</span>
</p>
```

**Comment by JoryHogeveen** [comment]
@jimtrue @ChristopheSeguinot 

I believe this is related to a the `limit` param. If you don't set a limit then the pagination doesn't know how to calculate the total pages.
https://github.com/pods-framework/pods/blob/main/classes/Pods.php#L3535-L3539

**Comment by sc0ttkclark** [comment]
Fixed via f917b0b881a2e778da4eda8d1c4ddbdf2bdb645a

ceil() was returning a float and the logic using it was a strict check against the integer 1.
